----------------------------------------------------------------
Thu Aug 10 14:07:55 CEST 2017:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): instance a816c00e-015d-cc0b-3e44-000006d6fb90 
on database directory C:\Users\avonva\Desktop\CatalogueBrowser\CatalogueBrowser\Database\MAIN_CATS_DB with class loader sun.misc.Launcher$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/avonva/Desktop/CatalogueBrowser/CatalogueBrowser/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=C:\Users\avonva\Desktop\CatalogueBrowser\CatalogueBrowser
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
Thu Aug 10 14:08:16 CEST 2017 Thread[Thread-3,5,main] Cleanup action starting
java.sql.SQLException: Database 'Database\PRODUCTION_CATS\CAT_MTX_DB\MTX_VERSION_8.11' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at catalogue.Catalogue.getConnection(Catalogue.java:2354)
	at import_catalogue.CatalogueSheetImporter.createDatabase(CatalogueSheetImporter.java:185)
	at import_catalogue.CatalogueSheetImporter.insert(CatalogueSheetImporter.java:168)
	at import_catalogue.SheetImporter.importData(SheetImporter.java:72)
	at import_catalogue.CatalogueWorkbookImporter.importCatalogueSheet(CatalogueWorkbookImporter.java:155)
	at import_catalogue.CatalogueWorkbookImporter.importWorkbook(CatalogueWorkbookImporter.java:72)
	at import_catalogue.CatalogueImporter.importXlsx(CatalogueImporter.java:200)
	at import_catalogue.CatalogueImporter.importXml(CatalogueImporter.java:180)
	at import_catalogue.CatalogueImporter.importEcf(CatalogueImporter.java:163)
	at import_catalogue.CatalogueImporter.makeImport(CatalogueImporter.java:67)
	at import_catalogue.CatalogueImporterThread.run(CatalogueImporterThread.java:67)
Caused by: ERROR XJ004: Database 'Database\PRODUCTION_CATS\CAT_MTX_DB\MTX_VERSION_8.11' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 27 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'Database\PRODUCTION_CATS\CAT_MTX_DB\MTX_VERSION_8.11' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at catalogue.Catalogue.getConnection(Catalogue.java:2354)
	at import_catalogue.CatalogueSheetImporter.createDatabase(CatalogueSheetImporter.java:185)
	at import_catalogue.CatalogueSheetImporter.insert(CatalogueSheetImporter.java:168)
	at import_catalogue.SheetImporter.importData(SheetImporter.java:72)
	at import_catalogue.CatalogueWorkbookImporter.importCatalogueSheet(CatalogueWorkbookImporter.java:155)
	at import_catalogue.CatalogueWorkbookImporter.importWorkbook(CatalogueWorkbookImporter.java:72)
	at import_catalogue.CatalogueImporter.importXlsx(CatalogueImporter.java:200)
	at import_catalogue.CatalogueImporter.importXml(CatalogueImporter.java:180)
	at import_catalogue.CatalogueImporter.importEcf(CatalogueImporter.java:163)
	at import_catalogue.CatalogueImporter.makeImport(CatalogueImporter.java:67)
	at import_catalogue.CatalogueImporterThread.run(CatalogueImporterThread.java:67)
============= end nested exception, level (1) ===========
Cleanup action completed
----------------------------------------------------------------
Thu Aug 10 14:08:16 CEST 2017:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): instance e0b94027-015d-cc0b-3e44-000006d6fb90 
on database directory C:\Users\avonva\Desktop\CatalogueBrowser\CatalogueBrowser\Database\PRODUCTION_CATS\CAT_MTX_DB\MTX_VERSION_8.11 with class loader sun.misc.Launcher$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/avonva/Desktop/CatalogueBrowser/CatalogueBrowser/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=C:\Users\avonva\Desktop\CatalogueBrowser\CatalogueBrowser
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Thu Aug 10 14:08:19 CEST 2017:
Shutting down instance e0b94027-015d-cc0b-3e44-000006d6fb90 on database directory C:\Users\avonva\Desktop\CatalogueBrowser\CatalogueBrowser\Database\PRODUCTION_CATS\CAT_MTX_DB\MTX_VERSION_8.11 with class loader sun.misc.Launcher$AppClassLoader@73d16e93 
----------------------------------------------------------------
Thu Aug 10 14:08:19 CEST 2017:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): instance 789c8155-015d-cc0b-3e44-000006d6fb90 
on database directory C:\Users\avonva\Desktop\CatalogueBrowser\CatalogueBrowser\Database\PRODUCTION_CATS\CAT_MTX_DB\MTX_VERSION_8.11 with class loader sun.misc.Launcher$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/avonva/Desktop/CatalogueBrowser/CatalogueBrowser/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=C:\Users\avonva\Desktop\CatalogueBrowser\CatalogueBrowser
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Thu Aug 10 14:10:26 CEST 2017:
Shutting down instance a816c00e-015d-cc0b-3e44-000006d6fb90 on database directory C:\Users\avonva\Desktop\CatalogueBrowser\CatalogueBrowser\Database\MAIN_CATS_DB with class loader sun.misc.Launcher$AppClassLoader@73d16e93 
